application/forms/Config/UserGroup/UserGroupBackendForm.php

```diff
@@ -190,4 +214,91 @@ public function onRequest()
             $this->populate($data);
         }
     }
+
+    /**
+     * Run the configured backend's inspection checks and show the result, if necessary
+     *
+     * This will only run any validation if the user pushed the 'backend_validation' button.
+     *
+     * @param   array   $formData
+     *
+     * @return  bool
+     */
+    public function isValidPartial(array $formData)
+    {
+        if (! parent::isValidPartial($formData)) {
```

Comment:
Please remove this call. Unlike for UserBackends it's not required here and results only in annoying validation errors when interacting with autosubmit elements.